app/models/gias_school.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: gias_schools
+#
+#  id         :uuid             not null, primary key
+#  name       :string           not null
+#  postcode   :string
+#  town       :string
+#  urn        :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+# Indexes
+#
+#  index_gias_schools_on_urn  (urn) UNIQUE
+#
+class GiasSchool < ApplicationRecord
+  validates :urn, presence: true
```

Comment:
Is it worth replicating the validations used in Publish?
https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/models/gias_school.rb

They appear to include `:address1`, `:town`, `:postcode`

Reply:
Probably, I'll make the change

Reply:
So, I think we don't want to validate too much of the gias data. Not yet at least. When it comes to address/postcode, we might use the `Easting` and `Northing` coordinates, not imported just yet(but in the CSV gias file) to determine the exact location of a school. 

So at the moment we are fine with importing schools that don't have a postcode or town or address.